src/sls_facades.js

```diff
@@ -0,0 +1,392 @@
+/**
+ * Embed a Sheets chart (indicated by the spreadsheetId and sheetChartId) onto
+ *   a page in the presentation. Setting the linking mode as 'LINKED' allows the
+ *   chart to be refreshed if the Sheets version is updated.
+ * @param {string} presentationId
+ * @param {string} spreadsheetId
+ * @param {string} sheetChartId
+ * @param {string} slidePageId
+ * @param {string} slideChartShape
+ * @return {*}
+ */
+function replaceSlideShapeWithSheetsChart(
+  presentationId, spreadsheetId, sheetChartId, slidePageId, slideChartShape) {
+const chartHeight = slideChartShape.getInherentHeight();
+const chartWidth = slideChartShape.getInherentWidth();
+const chartTransform = slideChartShape.getTransform();
+const presentationChartId = 'chart-test';
+const requests = [{
+  createSheetsChart: {
+    objectId: presentationChartId,
+    spreadsheetId: spreadsheetId,
+    chartId: sheetChartId,
+    linkingMode: 'LINKED',
+    elementProperties: {
+      pageObjectId: slidePageId,
+      size: {
+        width: {magnitude: chartHeight, unit: 'PT'},
+        height: {magnitude: chartWidth, unit: 'PT'},
+      },
+      transform: {
+        scaleX: chartTransform.getScaleX(),
+        scaleY: chartTransform.getScaleY(),
+        translateX: chartTransform.getTranslateX(),
+        translateY: chartTransform.getTranslateY(),
+        unit: 'PT',
+      },
+    },
+  },
+}];
+
+// Execute the request.
+try {
+  const batchUpdateResponse =
+      Slides.Presentations.batchUpdate({requests: requests}, presentationId);
+  console.log('Added a linked Sheets chart with ID: %s', presentationChartId);
+  slideChartShape.remove();
+  return batchUpdateResponse;
+} catch (err) {
+  // TODO (Developer) - Handle exception
+  console.log('Failed with error: %s', err.error);
```

Comment:
There's two things happening here, the error is being logged twice due to a merging issue (yikes!), and the logging format comes straight from the Sheets docs, but I found it to not be very informative regardless when using the debugger in the AppsScript editor. Will remove the duplicate line and add an issue to look into this, thank you for flagging.

Reply:
You might have seen this on other parts of the code, where this was being used before extracting it to the facades -library?- file so that any audit type could use it. A lot of these audits aren't being used anymore, so I figure we could always deprecate them and delete them.